src/constants.js

```diff
@@ -19,3 +19,17 @@ export const DAYS_OF_WEEK = [
   "Friday",
   "Saturday",
 ];
+
+export const ENGLISH_LEVEL = [
+  "Not required",
+  "Basic",
+  "Intermediate",
+  "Native",
+  "Fluent/Advance",
+];
+export const JOB_TYPE = ["Part-time", "Full-time", "Shift based"];
+export const SALARY_TYPE = ["Hourly", "Yearly"];
+
+export const WAGE_FILTER = [0, 5, 10, 18, 25, 32];
+export const JOB_POSTED_FILTER = ["Anytime", "Past 3 days", "Past week", "Past month"];
+export const BENEFIT = ["Medical", "Others"];
```

Comment:
here, I would also like to include only general things here. Put these filters definition to JobList sepecific file